File: c/Ex34.c
#include<stdio.h>
int main(int argc, char *argv[], int i, int a){
  
int numeros[10];
  printf("Digite os 10 numeros:\n"); 
  for(i=0; i<10; i++)
  {
    scanf("%i", &numeros[i]);
    if (numeros[i]>a)
    {
    	a=numeros[i];
	}
  } 
 	a=0;
  printf("\n\nOs numeros digitados foram:\n");
  for(i=9; i>=0; i--)
  {
      printf("%i \n", numeros[i]);
  }
  printf("\n\nO maior numero e: %i", a);   
  return 0;
}

File: python/Ex34.py
numeros = []
c=0
m=0
for i in range(0, 10, 1):
    x = int(input('Digite um numero: '))
    numeros.append (x)
print('\nOs numeros que você digitou foram:\n') 

for i in range(0, 10, 1):
    print(numeros[i])
    c=numeros[i]
    if (c>m):
        m=c
print ("\n O maior número do vetor é o:", m)